pom.xml

```diff
@@ -419,6 +427,18 @@
             <artifactId>junit-vintage-engine</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Would it make sense to group the Jupiter dependencies together?

Reply:
You mean adding them to a dependencyManagement? If so, I could, but it will make the pom file a little bit massive.

Reply:
No, no. I just meant have them together. One below the other, instead of having other dependencies in between.

Reply:
Ok, got it. But it looks like all the junit dependencies are grouped together, I did not see any other dependencies which is not JUnit related in between?

Reply:
Yeah, but it's Jupiter-api, then junit. then jupiter-engine, then junit-vintage etc. What I'm saying is get the Jupiter dependencies together.

Reply:
reorganize the order